Docs/Configuration.tex

```diff
@@ -3186,98 +3208,101 @@ \subsection{Boot Properties}\label{miscbootprops}
 
 \item
   \texttt{PickerMode}\\
   \textbf{Type}: \texttt{plist\ string}\\
   \textbf{Failsafe}: \texttt{Builtin}\\
   \textbf{Description}: Choose boot picker used for boot management.
 
-  Picker describes underlying boot management with an optional user interface
-  responsible for handling boot options. The following values are supported:
+  PickerMode describes the underlying boot management with an optional user
+  interface responsible for handling boot options.
+
+  The following values are supported:
 
   \begin{itemize}
   \tightlist
   \item \texttt{Builtin} --- boot management is handled by OpenCore, a simple
-  text only user interface is used.
+  text-only user interface is used.
   \item \texttt{External} --- an external boot management protocol is used
-  if available. Otherwise \texttt{Builtin} mode is used.
+  if available. Otherwise, the \texttt{Builtin} mode is used.
   \item \texttt{Apple} --- Apple boot management is used if available.
-  Otherwise \texttt{Builtin} mode is used.
+  Otherwise, the \texttt{Builtin} mode is used.
   \end{itemize}
 
-  Upon success \texttt{External} mode will entirely disable all boot management
-  in OpenCore except policy enforcement. In \texttt{Apple} mode it may additionally
-  bypass policy enforcement. See \hyperref[ueficanopy]{OpenCanopy} plugin
-  for an example of a custom user interface.
+  Upon success, the \texttt{External} mode will entirely disable all boot management
+  in OpenCore except for policy enforcement. In the \texttt{Apple} mode, it may
+  additionally bypass policy enforcement. Refer to the \hyperref[ueficanopy]{OpenCanopy}
+  plugin for an example of a custom user interface.
 
-  OpenCore built-in boot picker contains a set of actions chosen during the boot process.
-  The list of supported actions is similar to Apple BDS and in general can be accessed by
-  holding \texttt{action hotkeys} during boot process. Currently the following actions are
-  considered:
+  The OpenCore built-in boot picker contains a set of actions chosen during the boot process.
+  The list of supported actions is similar to Apple BDS and typically can be accessed by
+  holding \texttt{action hotkeys} during the boot process.
+
+  The following actions are currently considered:
 
   \begin{itemize}
   \tightlist
-  \item \texttt{Default} --- this is the default option, and it lets OpenCore built-in
-  boot picker to loads the default boot option as specified in
+  \item \texttt{Default} --- this is the default option, and it lets the built-in OpenCore
+  boot picker to load the default boot option as specified in the
   \href{https://support.apple.com/HT202796}{Startup Disk} preference pane.
-  \item \texttt{ShowPicker} --- this option forces picker to show. Normally it can be
-  achieved by holding \texttt{OPT} key during boot. Setting \texttt{ShowPicker} to
+  \item \texttt{ShowPicker} --- this option forces the boot picker to be displayed. This can
```

Comment:
We try to use `picker` when we talk about OpenCore boot menu and `boot picker` when we talk about Apple's one. I'd opt to be consistent with this convention.

Reply:
Used `OpenCore picker` and `Apple picker` respectively